config/locales/en/events.yml

```diff
@@ -23,5 +23,5 @@ en:
     title:
       self: Caseworker removed self from claim
       secondary: Caseworker removed from claim by %{display_name}
-  event/delete_adjustments:
-    title: Caseworker deleted all adjustments
```

Comment:
Was this meant to be removed?

Reply:
Ooh no, that looks like a merge/rebase error. Will fix